app/controllers/ops_controller/ops_rbac.rb

```diff
@@ -845,8 +859,9 @@ def process_roles(roles, task)
     process_elements(roles, MiqUserRole, task)
   end
 
-  def process_tenants(tenants, task)
-    process_elements(tenants, Tenant, task, "Tenant", "name")
+  def process_tenants(tenants, task, display_model = 'Tenant')
```

Comment:
@kbrock  - The problem is that we display Projects rather than the actual model when divisible=false.
You are right about the Tenant.find_by_id(Tenant.find_by_id..)... This was the same before this PR and I did not notice - the same is true for groups and users - I will address your comments and make changes for groups and users as well.